protocol/scripts/deploys/mainnet/v2/core/99-core-post-deployment.ts

```diff
@@ -0,0 +1,100 @@
+import { ethers } from 'hardhat';
+import { 
+  ensureExpectedEnvvars,
+  impersonateSigner,
+  mine
+} from '../../../helpers';
+import { 
+  connectToContracts,
+  getDeployedContracts
+} from '../contract-addresses';
+import { ITempleStrategy, TempleERC20Token__factory } from '../../../../../typechain';
+import { Signer } from 'ethers';
+
+
+async function impersonateAndFund(owner: Signer, address: string, amount: number): Promise<Signer> {
```

Comment:
This should be the *actual* deploy script that we'll actually end up when pushing out these contracts for real in mainnet.

So we shouldn't have an impersonateSigner here.

If you need to do any impersonation to get some test tokens, then that should be done in your `999-localhost-fork-test.ts`